assembl/static2/js/app/utils/translations.js

```diff
@@ -1538,7 +1543,12 @@ const Translations = {
         postSuccess: "Thanks for your participation. Your proposal has been sent!",
         postToBeValidated:
           "Your contribution has been saved. To insure the quality of the debate, it will be visible to all participants once it has been reviewed by the animation team.",
-        endPhase: "The \"%{closedPhaseName}\" phase is closed."
+        endPhase: "The \"%{closedPhaseName}\" phase is closed.",
+        postsOrder: {
+          chronologicalTop: "Chronological points of view",
```

Comment:
Not sure I'd use 'points' here but I suspect it's already the standard term in assembl. We would normally just call it a post.

Reply:
You may be right, but the labelling was specified in the ticket.